modules/openapi-generator/src/test/resources/3_0/all_of_with_additional_properties.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.0.2
+info:
+  version: 1.0.0
+  title: Example
+  license:
+    name: MIT
+servers:
+  - url: http://api.example.xyz/v1
+paths:
+  /person/display/{personId}:
+    get:
+      parameters:
+        - name: personId
+          in: path
+          required: true
+          description: The id of the person to retrieve
+          schema:
+            type: string
+      operationId: list
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Person"
+components:
+  schemas:
+    Person:
+      description: A representation of a person
+      type: object
+      properties:
+        name:
+          type: string
+    Adult:
+      description: A representation of an adult
+      additionalProperties: {}
+      allOf:
+      - $ref: '#/components/schemas/Person'
+      - type: object
+        properties:
+          address:
+            type: string
```

Comment:
Could you move these into the python-experimental samples spec?
That way we can test instantiating these classes in python.

Reply:
I remove this code.